test/e2e/data/infrastructure-aws/kustomize_sources/nested-multitenancy-clusterclass/cluster-template.yaml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
We can define a base ClusterClass template and use it in all ClusterClass tests considering more tests are coming up, this will reduce replication. Not needed for this PR.

Reply:
Agreed. As we have discussed on the issue, that is the ultimate plan. As I am building a ClusterClass and verifying e2e, it is a little hard to do both at the same time and hence focusing on building each portion of ClusterClass.

After a few tests, will refactor to combine ClusterClasses.